yarn.lock

```diff
@@ -95,7 +95,11 @@ __metadata:
     prop-types: "npm:^15.7.2"
     react-is: "npm:^16.9.0"
   peerDependencies:
+    react-dom: ^16.8.3
```

Comment:
Wondering why `react` isn't listed here. It's listed in the manifest of `react-redux`.

Reply:
That's strange - removing the `react-redux` entry and running `yarn install` again makes it appear as expected 🤔